lxd/container_lxc.go

```diff
@@ -6068,6 +6068,23 @@ func (c *containerLXC) Exec(command []string, env map[string]string, stdin *os.F
 	cmd.Stdout = logFile
 	cmd.Stderr = logFile
 
+	// Mitigation for CVE-2019-5736
+	useRexec := false
+	if c.expandedConfig["raw.idmap"] != "" {
+		err := allowedUnprivilegedOnlyMap(c.expandedConfig["raw.idmap"])
+		if err != nil {
+			useRexec = true
+		}
```

Comment:
So this still executes for all unprivileged containers not just ones where 0 is mapped to 0? I'm fine with that just checking.

Reply:
No, this should only set reExec if allowedUnprivilegedOnlyMap returns an error and it's only supposed to do that if root is mapped.

It's the same function that we use when LXD_UNPRIVILEGED_ONLY is set on Chromebooks.

Reply:
Sweet.